config/initializers/i18n-js.rb

```diff
@@ -38,5 +38,16 @@ def self.translations
       all_translations
     end
 
+    class FallbackLocales
```

Comment:
Opening up existing class. Here's the original https://github.com/fnando/i18n-js/blob/master/lib/i18n/js/fallback_locales.rb